.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - name: Install unzip
         run: apt-get update && apt-get install -y unzip
       - name: Run tests
-        run: cp .env.testing.template .env.testing && make test
```

Comment:
Interesting, this is why `git diff --quiet Package.resolved` didn't work:

<img width="670" alt="CleanShot 2021-05-20 at 09 26 53@2x" src="https://user-images.githubusercontent.com/65520/118937329-b548ce80-b94d-11eb-996f-1ac42296a42d.png">

I'll see if I can bump the base image with a newer version.